apps/data-types.md

```diff
@@ -577,6 +577,7 @@ This object is used for the attributes of basePrice, depositPrice, ecoParticipat
 | sfUrl | string | The link to the storefront URL of the product. |
 | productNumber | string | The product number. |
 | isProductNumberVisible | boolean | Indicates if the product number is displayed in the storefront. |
+| isNew | boolean | Indicates if the product should be displayed as in the storefront. |
```

Comment:
I would suggest to slightly change the wording. Furthermore I added the "word" as it was missing.
```suggestion
| isNew | boolean | Indicates if the product is displayed as new in the storefront. |
```

Reply:
@dkluever22 you are totally right it must also be adapted in '_raml/apps/search/post_search_products_response.json'